### PR TITLE
Joint verse

### PR DESCRIPTION
**Pull request by vipinpaul**
Joint verse(s) support enabled, the user can join or remove any verse(s) manually as well as the user can import and export the files with Joint verse(s). #78

### CODE REVIEW

File: src/components/TranslationPanel.js [hunk 1]
```diff
@@ -135,6 +203,22 @@ class TranslationPanel extends React.Component {
 				{tIns || tDel ? <div style={{textAlign: "center"}}><span style={{color: '#27b97e', fontWeight: 'bold'}}>(+) <span id="tIns">{tIns}</span></span> | <span style={{color: '#f50808', fontWeight: 'bold'}}> (-) <span id="tDel">{tDel}</span></span></div> : "" }
 				<div id="input-verses" className={`col-12 col-ref verse-input ${AutographaStore.scriptDirection.toLowerCase()} ${tIns || tDel ? 'disable-input' : ''}`} dir={AutographaStore.scriptDirection} style={{pointerEvents: tIns || tDel ? 'none': ''}}>{verseGroup}</div>
 				</div>
+				<ContextMenu id={"false"}>
+					<MenuItem
+						data={{ action: 'Remove Joint' }}
+						onClick={this.removeJoint}
+					>
+						Unjoin this verse
```
Comment (joelthe1) [inline src/components/TranslationPanel.js]: Translations needed here.

File: src/components/TranslationPanel.js [hunk 2]
```diff
@@ -135,6 +203,22 @@ class TranslationPanel extends React.Component {
...
+					</MenuItem>
+				</ContextMenu>
+				<ContextMenu id={"true"}>
+					<MenuItem
+						data={{ action: 'Joint' }}
+						onClick={this.addJoint}
+					>
+						Join to preceeding verse
```
Comment (joelthe1) [inline src/components/TranslationPanel.js]: And here.

File: src/components/TranslationPanel.js (hunk 1 above)
Comment (vipinpaul) [reply src/components/TranslationPanel.js]: Yeah, I have forgooten that.

File: src/components/TranslationPanel.js (hunk 2 above)
Comment (vipinpaul) [reply src/components/TranslationPanel.js]: Will do it.

**Pull request merged**